src/content/components/code-snippet/code.md

```diff
@@ -3,13 +3,11 @@ desc: <span>Code snippets</span> are small blocks of reusable code that can be i
 variations:
   code-snippet:
   - Code Snippet
-  code-snippet--multi:
+  code-snippet--terminal:
```

Comment:
Seems that the original code here is in line with [latest `v9` branch of `carbon-components`](https://github.com/carbon-design-system/carbon-components/blob/v9/src/components/code-snippet/code-snippet.config.js) - Good to ensure the following:

* `carbon-components` directory is located as a sibling of `design-system-website`
* `carbon-components` has latest `v9` branch checked out
* Running `yarn build` in `carbon-components`, and then doing:

```sh
> cd /path/to/design-system-website
> rm -Rf node_modules
> yarn install
```